packages/components/molecules/f-searchbox/CHANGELOG.md

```diff
@@ -40,20 +55,37 @@ v4.0.0-beta.14
 
 v4.0.0-beta.13
 ------------------------------
-*January 5, 2021*
+*January 05, 2021*
 
 ### Fixed
-- DK & NO field options - provides component with custom attributes, supplied via tenant config `addressField`.
+- DK & NO field options - provides component with custom attributes, 
+supplied via tenant config `addressField`.
 - DK & NO uses the `where` form value now.
-- Google test from failing silently in CI.
 
 ### Added
 - url-loader so we can inline the `powered by google` logo as a base64 URI.
-- Tests to helper.js.
+
+
+Latest (add to next release)
+------------------------------
+*December 30, 2020*
 
 ### Changed
 - Updated config for latest `sass-loader`.
 - Switches import in `common.scss` in line with fozzie v5-beta.
+
+### Fixed
+- Google test from failing silently in CI.
+
+
+Latest (to be added to next release)
```

Comment:
Don't think this heading should be here?

Reply:
Fixed.